### DISCUSSION TITLE
Add additional header parameter for captureException method

### DISCUSSION THREAD

**Discussion by OmniRafa** [getsentry/sentry-dart#1528] [Q&A]
Hi all, I would like to add an additional header parameter for the outgoing request that the captureException method creates. 
The problem is currently, the SentryServer is behind a securitywall . For the request to get through, it must contain a token. Every request to the server should contain this token. Is this possible?

**Comment by marandaneto** [comment]
@OmniRafa Right now this is not possible unless you make a custom `transport`.
You have to write and set your own `transport` into the `options.transport` variable.
This is probably easy if you duplicate most of the `HttpTransport` class.
Another option is that you add this capability to the SDK, by extending the `SentryOptions`, and `HttpTransport` classes, so `HttpTransport#_buildHeaders` tale into consideration an extra list of headers that come from the Client.
Are you willing to open a PR?
The Java SDK does something similar.

**Reply by OmniRafa** [reply]
Thank you very much for the detailed and helpful answer :) I will evaluate the effort with our customer and then we see how we will proceed. If we decide to go for the PR, I will post it here

**Reply by marandaneto** [reply]
Similar issue https://github.com/getsentry/sentry-dart/issues/975
